scripts/deploy/deploy-webapi.ps1

```diff
@@ -56,8 +60,39 @@ if ($LASTEXITCODE -ne 0) {
     exit $LASTEXITCODE
 }
 
+# Check if DeploymentSlot parameter was passed
+if ($DeploymentSlot) {
+
+    Write-Host "Checking if slot $DeploymentSlot exists for '$webappName'..."
+    $availableSlots = az webapp deployment slot list --resource-group utverse-devl --name app-utverse-devl-webapi | ConvertFrom-JSON | Select-Object -Property Name
```

Comment:
utverse-devl???  That should be replaced by the param to this script! Same at line 80.